lib/ip_blocklist_manager.js

```diff
@@ -31,12 +31,49 @@ module.exports = function (log, config) {
     })
 
     return Promise.all(loadedLists)
+      .then(function () {
+
+        // Save the absolute path of each loaded blocklist for
+        // later use.
+        self.loadedLists = self.ipBlocklists.map(function (list) {
+          return list.filePath
+        })
+      })
   }
 
   IPBlocklistManager.prototype.clear = function () {
     this.ipBlocklists = []
   }
 
+  IPBlocklistManager.prototype.shouldBlock = function (ipAddress) {
+    var self = this
+    const blockInfo = self.contains(ipAddress)
+
+    if (blockInfo.found) {
+
+      // When overriding the blocklist blocking rules, you must override
+      // rules for all the lists.
+      const listOverrideRules = config.ipBlocklist.listsOverrideRules
+      if (listOverrideRules && listOverrideRules.length > 0 &&
+        self.loadedLists.length === listOverrideRules.length) {
+
+        // Lookup the overrided blocklist config and block
+        // request if set to `true`
+        return blockInfo.listPaths.some(function (listPath) {
+          const listIndex = self.loadedLists.indexOf(listPath)
+
+          return (listOverrideRules[listIndex] === 'true')
```

Comment:
Not ideal, but from https://github.com/mozilla/node-convict/blob/master/lib/convict.js#L327, the passed array properties will be stored as `string` and not `boolean`.